### ISSUE TITLE
Support dark mode

### ISSUE THREAD

**Issue by sjjhsjjh** [dasher-project/redash#59]
The Dasher Version Six user interface (UI) should support dark mode.

The control panel and other parts of the UI that are like a web page should have a CSS colour scheme that changes based on the current preference, for example by including a `@media (prefers-color-scheme: dark)` query. The pagebuilder.css file from the Captive Web View project already has colour scheme that is mostly suitable.

The zooming part of the UI should also have different colours in light and dark modes. One approach could be as follows.
-   In light mode, same colours as current.
-   In dark mode, no fill colours but show white outlines and white text.

**Comment by willwade** [comment]
Yeah. I think this is wise. Particularly given its now pretty integral to most OS’ 

@sjjhsjjh  we have a colour theme for dasher. Do we make this part of that system ? Like change both overall UI and interface ?
